pytest.ini

```diff
@@ -0,0 +1,7 @@
+[pytest]
+env =
+    GITHUB_PR_ID=1
+    REPO_USERNAME=daniel
+    TOKEN=incdsc98er
```

Comment:
Is this a real token?

Reply:
No, no, no, my darling. Just some random letters.

Reply:
An interesting fact: if I had actually sent the token (as I have done in one of the previous commits), GitHub automatically recognises this as a risk and revokes it.